submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,104 @@
+const arrOfUnits = ["img/cartman.jpg",
+    "img/chief.jpg",
+    "img/sten.jpg",
+    "img/kyle.jpg",
+    "img/kenny.jpg"
+];
+const imgCover = `img/cover.jpg`;
+const gameField = document.querySelector("#gameField");
+let isLocked = false;
+let firstCard;
+let secondCard;
+let isSecondCard = false;
+let unopenedCards;
+
+//feed him array & any even number
+const duplicateElements = function(array, times) {
+    return array.reduce((res, current) => {
+        return res.concat(Array(times).fill(current));
+    }, []);
+};
+
+const shuffle = function(arr) {
+    const result = [];
+    while (arr.length > 0) {
+        let random = Math.floor(Math.random() * arr.length);
+        let elem = arr.splice(random, 1)[0];
+        result.push(elem);
+    }
+    return result;
+};
+
+const checkCard = function({ target }) {
+    if (!isLocked) {
+        if (isSecondCard) {
+            isLocked = true;
+            secondCard = target.closest("div");
```

Comment:
That is why your field starts 'mirroring' if clicked. You are searching for a `div`. Your field is `div` too. `.closest()` works with any selectors, so you can search for `.flipper` instead.

Reply:
fixed it.